components/google-cloud/google_cloud_pipeline_components/experimental/forecasting/validate/component.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Forecasting Validation
+inputs:
+- {name: input_tables, type: String}
+- {name: validation_theme, type: String}
+implementation:
+  container:
+    image: us-docker.pkg.dev/vertex-ai/time-series-forecasting/forecasting:prod
```

Comment:
just to be sure this container is publicly averrable is that correct? Also you may want to consider using a version number in the tag tie different releases to different containers however this is not a blocker for experimental.

Reply:
Right at this moment it is only experimental and there is no other versions. I'll probably create different images for V1, V2 etc in the future so that they can coexist.

Reply:
Will `:prod` always be backward compatible?

Right now this could run the risk of a user pulling the latest `prod` version with an older version of the SDK.

Reply:
Yes :prod will be backwards compatible. Next in google3 I will implement e2e test using the sdk on google3 in the container release workflow to guard any incompatible changes.